website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -1,94 +1,106 @@
+HDS organizes semantic tokens into **element** categories with **roles** appended: 
 
-- **Foreground** - For elements such as text, links, statuses, and icons.
-- **Border** - For borders on components, containers, or dividers.
+- **Foreground** - For text, link, status, and icon elements
+- **Border** - For borders (or strokes) on components, containers, or dividers
 - **Surface** - For the background (or surface) of a component or container
 - **Page** - For page backgrounds
 
-Examples of **role** naming conventions include: 
+Examples of **role** in naming include: 
 
 - Strong
 - Primary
 - Faint
 - Action (not to be used outside of context)
 - Disabled (not to be used outside of context)
 
-### Foreground colors
+### Foreground
 
-Foreground colors are used for elements such as text, links, and icons.
+Foreground colors are used for decorative and informational elements such as text, links, and icons.
 
 Some common examples of semantic foreground colors include:
 
 - `Foreground/Strong` for headings and secondary links
 - `Foreground/Primary` for body text 
 - `Foreground/Faint` for less prominent text and UI elements.
-- `Foreground/Action` for primary call to action such as links
+- `Foreground/Action` for primary calls to action such as links
 
 ![Samples of text with their color tokens labelled](/assets/foundations/color/colors-foreground-examples.png)
 
-Use status foreground colors to help contextualize responses from user actions or to indicate status within a UI. These colors should be used sparingly and within the context of specific components like [Alerts](/components/alert#color), [Toasts](/components/toast#color), or [Badges](/components/badge#color).
+Status foreground colors contextualize responses from user actions or indicate status within a UI. These colors should be used sparingly and within the context of specific components, for example:
+
+- [Alerts](/components/alert#color)
+- [Toasts](/components/toast#color)
+- [Badges](/components/badge#color).
 
 ![Sample positive, warning, and error status badges with their associated foreground color tokens labelled.](/assets/foundations/color/colors-status-examples.png)
 
-### Border colors
+### Border
 
-Some common examples of border colors include:
+Some examples of border colors include:
 
 - `Border/Strong` for secondary Button border
 - `Border/Primary` for Card border or divider
 - `Border/{Status color}` for Alert borders
 
 ![Sample borders applied on a button, card, and alert with the border color tokens labelled](/assets/foundations/color/colors-border-examples.png)
 
-### Surface colors
+### Surface
 
-Use surface colors for the background (or surface) of a component or container. 
+Surface colors are intended to be used for the background (or surface) of a component or container. 
 
-Some common examples of surface colors include:
+Examples of surface colors include:
 
 - `Surface/Strong` for the neutral Badge.
```

Comment:
```suggestion
- `Surface/Strong` for the neutral Badge
```

Reply:
Suggestion: Just because in most lists you don't have periods at the end of the list item